blockchain/evm.go

```diff
@@ -45,6 +45,8 @@ type ChainContext interface {
 
 	// GetHeader returns the hash corresponding to their hash.
 	GetHeader(common.Hash, uint64) *types.Header
+
+	Config() *params.ChainConfig
```

Comment:
Is it unnecessary because we now use ChainReader?

Reply:
Thank you fixed. https://github.com/kaiachain/kaia/pull/121/commits/5eeba8bf39278d306526d4f23dd0524a820c436b